### PR TITLE
Add more test cases (#1)

### PR DESCRIPTION
**Pull request by dieu-heureka**

### CODE REVIEW

File: PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "a6b35002-048d-4c78-8a39-85ddb1a7a41f",
```
Comment (LoxxessWetzel) [inline PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json]: Best practice is to exclude the update of this id. It's changed every time you save it.

File: PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json [hunk 2]
```diff
@@ -159,15 +159,91 @@
...
 							"script": {
 								"exec": [
-									"pm.test(\"Category does not exist\", function () {\r",
-									"    pm.response.to.have.status(404);\r",
-									"    pm.expect(pm.response.json().message).to.eql(\"Category zzzyyyccc does not exist\");\r",
+									"pm.test(\"Article was created\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"\r",
+									"$articleId = pm.collectionVariables.get(\"ARTILE_ID\");\r",
```
Comment (LoxxessWetzel) [inline PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json]: Please rename ARTILE_ID to ARTICLE_ID

File: PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json [hunk 3]
```diff
@@ -227,6 +449,706 @@
...
+			"name": "Location",
+			"item": [
+				{
+					"name": "Create Location",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Stock was updated.\", function () {\r",
```
Comment (LoxxessWetzel) [inline PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json]: The stock should imho not be mentioned here. May be location is correct?

**Review by LoxxessWetzel — CHANGES_REQUESTED** [review]
Please review my comments.

File: PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json (hunk 1 above)
Comment (dieu-heureka) [reply PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json]: Yes, updated

File: PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json (hunk 2 above)
Comment (dieu-heureka) [reply PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json]: Updated, thanks

File: PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json (hunk 3 above)
Comment (dieu-heureka) [reply PostmanTests/Loxxess.Lx1.LosConnector.postman_collection.json]: Yes, it should be location. Thanks

**Review by ThomasWetzel — APPROVED** [review]

**Review by ThomasWetzel — APPROVED** [review]

**Pull request merged**